cyoa/app/views/rounds/ron_index.html.haml

```diff
@@ -0,0 +1,8 @@
+WHICH ONE DID RON SWANSON SAY?
+- @questions.shuffle.each do |question|
+  %p=question.trivia_piece
+  =form_for @current_round, data: {round_number: 'cats'} do |f|
+    =f.hidden_field :answer, value: question.id
+    =f.submit value: "Ron Said This", data: {submit_answer: 'blah'}
+
+%div{data:{result_holder: 'display'}}
```

Comment:
Hash attribute should start with one space after the opening brace<br>Hash attribute should end with one space before the closing brace